packages/ui-kit/src/components/DataGrid/icons.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+// TODO: move those icons to shared
```

Comment:
Shall we actually move the icons to https://github.com/propeldata/ui-kit/tree/main/packages/ui-kit/src/components/Icons?